index.html

```diff
@@ -199,7 +198,8 @@ <h3>## Diversity</h3>
         <br />
 
         <p>
-          We'll inform you as soon as the application phase for diversity tickets will begin.
+          The application phase has started. Use the <a target= "_blank" href = "https://docs.google.com/forms/d/e/1FAIpQLSdRY_j_snQky14Szoe9M3CztqQ4_AJcaovt8ebF3Px2a8-H_g/viewform">diversity form</a>
```

Comment:
you can add a line break before the link

```suggestion
          The application phase has started. Use the
          <a target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSdRY_j_snQky14Szoe9M3CztqQ4_AJcaovt8ebF3Px2a8-H_g/viewform">diversity form</a>
```

also note there should not be spaces before & after the `=` sign on HTML attributes